manim/mobject/geometry.py

```diff
@@ -795,6 +814,30 @@ def __init__(self, **kwargs):
 
 
 class ArrowTip(VMobject):
+    r"""Base class for arrow tips.
+
+
+    Examples
+    --------
+
+    Cannot be used directly, only intended for inheritance::
+        >>> tip = ArrowTip()
+        Traceback (most recent call last):
+        ...
+        NotImplementedError: Has to be implemented in inheriting subclasses.
+
+    Instead, use it like this::
+        >>> class MyCustomArrowTip(ArrowTip, RegularPolygon):
+        ...     def __init__(self, **kwargs):
+        ...         RegularPolygon.__init__(self, n=5, **kwargs)
+        ...         self.set_width(self.length)
+        ...         self.set_height(self.length, stretch=True)
+        >>> arr = Arrow(np.array([0, 0, 0]), np.array([1, 1, 0]),
+        ...             tip_shape=MyCustomArrowTip)
+        >>> isinstance(arr.tip, RegularPolygon)
+        True
+    
```

Comment:
Suggestion: add a "See also" section that points to all the pre-defined classes

Reply:
List added in 5528cc30a351ff57a5b06f3ab085620148cc2fe8